action.yml

```
@@ -187,11 +187,11 @@ runs:
         };
         export -f tf_convert_plan_to_json
         mapfile -t planfiles_json < <( find . -type f -name ${{ inputs.terraform-plan-filename }} | parallel 'tf_convert_plan_to_json {}' )
+        delimiter="$(openssl rand -hex 8)"
         content=$(printf "%s\n" "${planfiles_json[@]}")
-        content="${content//'%'/'%25'}"
-        content="${content//$'\n'/'%0A'}"
-        content="${content//$'\r'/'%0D'}"
-        echo "terraform_planfiles_json=${content}" >> $GITHUB_OUTPUT
+        echo "terraform_planfiles_json=${delimiter}" >> $GITHUB_OUTPUT
```

```suggestion
        echo "terraform_planfiles_json<<${delimiter}" >> $GITHUB_OUTPUT
```
laut anleitung muss es so sein